client/stylesheets/_toCat.scss

```diff
@@ -2085,6 +2100,7 @@ i:hover {
 .helpThree > li {
     width: 28%;
     display: inline-block;
+    vertical-align: top !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces